corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -641,6 +644,25 @@ hqDefine('geospatial/js/models', [
             updateSelectedSavedPolygonParam();
         };
 
+        self.deleteSelectedPolygonFilter = function () {
+            $.ajax({
+                type: 'DELETE',
+                url: geoJSONUrl,
+                dataType: 'json',
+                data: JSON.stringify({'polygon_id': self.selectedSavedPolygonId()}),
+                contentType: "application/json; charset=utf-8",
+                success: function (ret) {
+                    if (!ret.success) {
+                        return alertUser.alert_user(message, 'danger');
```

Comment:
what is the message here?

Reply:
its defined later, that won't cause an error here?

Reply:
Good catch. Resolved in https://github.com/dimagi/commcare-hq/pull/34717/commits/66e56f19acc1bd8a5fafa7ac60221fe19ba770ea